src/box/lua/config/instance_config.lua

```diff
@@ -1092,21 +948,7 @@ return schema.new('instance_config', add_descriptions(schema.record({
                         type = 'string',
                     })),
                 }),
-            }, {
-                validate = function(data, w)
-                    if next(data) == nil then
-                        w.error('An URI should have at least one field')
-                    end
-                    -- If a password is set, a login must also be specified.
-                    if data.password ~= nil and data.login == nil then
-                        w.error('Password cannot be set without setting login')
-                    end
-                    -- If a params is set, an uri must also be specified.
-                    if data.params ~= nil and data.uri == nil then
-                        w.error('Params cannot be set without setting uri')
-                    end
-                end,
-            }),
+            }, {}),
```

Comment:
Same, `, {}` is redundant, we can just drop the argument.

Reply:
The same for other such places.

Reply:
Dropped.